CONTRIBUTING.md

```diff
@@ -28,8 +28,7 @@ Before each big release, translations are pulled in from POEditor.
 
 ## Helping mappers by solving their notes
 
-- ☑️ requires no programming knowledge
-- ⬜️ requires no OpenStreetMap knowledge
+- Requires **no programming knowledge**!
```

Comment:
Yeah but, why not write _what_ it requires instead of what it _not_ requires??

Reply:
Because users are interested in what they can accomplish, even though they may not know something, rather than in what they cannot accomplish. Always write/say it in a positive way… :wink:

Reply:
Rather avoid double negatives and be straightforward.